.gitignore

```diff
@@ -145,4 +145,7 @@ dmypy.json
 *oresat_dxwifi/camera/lib/*
 
 # Camera Code: Output from camera
-oresat_dxwifi/camera/data/*
\ No newline at end of file
+oresat_dxwifi/camera/data/*
+
+# Transmission: Libdxwifi Python Bindings
+!oresat_dxwifi/transmission/tx_module.so
```

Comment:
We generally shouldn't include something that we build. Why are we tracking a compiled library as part of the git repo?

Reply:
What should we do instead?

Reply:
have the user build the .so file when they build everything else

Reply:
```
debian@cs-capstone-bbb:~/src/oresat-libdxwifi$ cd build && cmake .. && make && cd ..
...
-- Found pybind11: /usr/include (found version "2.6.2" )
...
[ 81%] Built target tx
[ 83%] Building CXX object dxwifi/tx/CMakeFiles/tx_module.dir/tx_wrapper.cpp.o
[ 85%] Linking CXX shared module ../../../bin/Debug/tx_module.so
...
[100%] Built target decode
```

Reply:
Wouldn't the defeat the purpose of the bindings? If we need to install libdxwifi anyways, we could just subprocess it since we have to anyways. Is there a way to host the library somewhere and include it as a dependency or something? Alternatively, we could just do what Ryan suggested and move libdxwifi here. I don't really have any preferences, I genuinely thought this is why we even wanted them in the first place, for convenient integration/easier build.

Reply:
I didn't say anything about installation; I'm talking about build. When you build libdxwifi currently, it creates the executable binaries (e.g., `tx`, `rx`) and also the `tx_module.so` file. Then it's your choice what to do with those.

It's not a good idea to include in a git repo a blob that's already compiled. It should be compiled on the target machine as needed.

Ideally we should have a libdxwifi build option that lets you choose whether to create the `tx_module.so`, but currently we do not.

Reply:
Wait a second -- this isn't libdxwifi, this is oresat-dxwifi-software. Hmm, let me think.